h2o-extensions/xgboost/src/test/java/hex/tree/xgboost/XGBoostVarImpComparisonTest.java

```diff
@@ -0,0 +1,112 @@
+package hex.tree.xgboost;
+
+import hex.SplitFrame;
+import hex.genmodel.utils.DistributionFamily;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import water.DKV;
+import water.Key;
+import water.Scope;
+import water.TestUtil;
+import water.fvec.Frame;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertNull;
+
+@RunWith(Parameterized.class)
+public class XGBoostVarImpComparisonTest extends TestUtil {
+
+    @BeforeClass
+    public static void setup() {
+        stall_till_cloudsize(1);
+    }
+
+    @Parameterized.Parameters(name = "XGBoost(booster={0},distribution={1},response={2}")
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][]{
+            {"gbtree", "AUTO", "AGE"},
+            {"gbtree", "bernoulli", "CAPSULE"},
+            {"gbtree", "multinomial", "CAPSULE"},
+            {"gbtree", "gaussian", "AGE"},
+            {"gbtree", "gamma", "AGE"},
+            {"gbtree", "poisson", "AGE"},
+            {"gbtree", "tweedie", "AGE"},
+            {"dart", "AUTO", "AGE"},
+            {"dart", "bernoulli", "CAPSULE"},
+            {"dart", "multinomial", "CAPSULE"},
+            {"dart", "gaussian", "AGE"},
+            {"dart", "gamma", "AGE"},
+            {"dart", "poisson", "AGE"},
+            {"dart", "tweedie", "AGE"},
+            {"gblinear", "AUTO", "AGE"},
+            {"gblinear", "bernoulli", "CAPSULE"},
+            {"gblinear", "multinomial", "CAPSULE"},
+            {"gblinear", "gaussian", "AGE"},
+            {"gblinear", "gamma", "AGE"},
+            {"gblinear", "poisson", "AGE"},
+            {"gblinear", "tweedie", "AGE"}
+        });
+    }
+
+    @Parameterized.Parameter
+    public String booster;
+
+    @Parameterized.Parameter(1)
+    public String distribution;
+
+    @Parameterized.Parameter(2)
+    public String response;
+
+    @Test
+    public void testImportancesAreSame() {
+        Scope.enter();
+        try {
+            Frame tfr = Scope.track(parse_test_file("./smalldata/prostate/prostate.csv"));
```

Comment:
there already is a version of the prostate dataset that is split to test and train, it is imho better for reproducibility of results